docs/APIRef.DeviceObjectAPI.md

```diff
@@ -80,37 +82,36 @@ Detox uses [AppleSimUtils](https://github.com/wix/AppleSimulatorUtils) to implem
 Launches the app with the specified URL to test your app's deep link handling mechanism.
 
 ```js
-await device.launchApp({url: url});
-await device.launchApp({url: url, newInstance: true}); //Launch a new instance of the app
-await device.launchApp({url: url, newInstance: false}); //Reuse existing instance
+await device.launchApp({url});
+await device.launchApp({url, newInstance: true}); // Launch a new instance of the app
+await device.launchApp({url, newInstance: false}); // Reuse existing instance
 ```
-This will launch the app from background and handle the deep link.
 
-Read more in [here](APIRef.MockingOpenWithURL.md).
+Read more [here](APIRef.MockingOpenWithURL.md). Go back to subsection 1 to read about the full effect of the `newInstance` argument.
```

Comment:
> Go back to subsection 1 to read about the full effect of the `newInstance` argument.

This is unnecessary for every section. People should read the guide if they want to understand how things work.

Reply:
You're right, yet this gives a certain cue as to what the reader should pay attention to (the final behavior depends on `newInstance`). I've found it confusing a bit myself, reading this at first.